src/sentry/static/sentry/app/components/helpSearch.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t, tn} from 'app/locale';
+import Search from 'app/components/search';
+import HelpSource from 'app/components/search/sources/helpSource';
+import SearchResult from 'app/components/search/searchResult';
+import SearchResultWrapper from 'app/components/search/searchResultWrapper';
+import space from 'app/styles/space';
+import {IconWindow} from 'app/icons';
+
+type HelpResult = Parameters<
+  React.ComponentProps<typeof HelpSource>['children']
+>[0]['results'][0];
+
+type ResultItemProps = HelpResult & {
+  highlighted: boolean;
+  itemProps: any;
+};
+
+const renderResult = ({item, matches, itemProps, highlighted}: ResultItemProps) => {
+  const sectionHeading =
+    item.sectionHeading !== undefined ? (
```

Comment:
```suggestion
    item.sectionHeading ? (
```

can `sectionHeading` also be `null`?

Reply:
No it will only be undefined or a string.